ui/src/main/js/application/auth/SamlForm.js

```diff
@@ -90,6 +97,20 @@ const SamlForm = ({ csrfToken, errorHandler, readonly, displayTest, fileDelete,
                 submitLabel="Save SAML Configuration"
                 testLabel="Test SAML Configuration"
             >
+                <LabeledField label={importBlackDuckSSOConfigLabel} description={importBlackDuckSSOConfigDescription}>
+                    <div className="d-inline-flex p-2">
```

Comment:
_Note: this isn't necessary, more of a nitpick_
We should utilize the reactjss library for the classname.  I'd love to phase out these bootstrap classes.  Here's what I would do:
After line 30, add another class:
```
fillForm: {
        padding: '0.5rem',
        display: 'inline-flex'
}
```
And then on this line (line 101), set the className to be:
`<div className={classes.fillForm}>`  
  
In this instance, `classes` references the variable on line 34.  That variable references the styles object that we just added the class, `fillForm`, on line 30.  So by calling `classes.fillForm` you are calling the class fillForm from above above.  Again this is a total nitpick by me.  Feel free to disregard or reach out if you have any questions.